src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -39,25 +60,44 @@ limitations under the License.
     <attribute side="server" code="0x0003" define="CURRENT_MODE"    type="INT8U"                                            writable="false" optional="false" isNullable="false" reportable="true">CurrentMode</attribute>
     <attribute side="server" code="0x0004" define="START_UP_MODE"   type="INT8U"                                            writable="true" optional="true"  isNullable="true">StartUpMode</attribute>
     <attribute side="server" code="0x0005" define="ON_MODE"         type="INT8U"                                            writable="true"  optional="true" isNullable="true">OnMode</attribute>
 
-    <!-- Test Commands -->
+    <!-- Commands -->
     <command source="client" code="0x00" name="ChangeToMode" optional="false">
       <description>
-        On receipt of this command, if the NewMode field matches the Mode field in an entry of the SupportedModes list, the server SHALL set the CurrentMode attribute to the NewMode value, otherwise, the server SHALL respond with an INVALID_COMMAND status response.
+        If the NewMode field doesn't match the Mode field in an entry of the SupportedModes list the server SHALL respond with an INVALID_COMMAND status response.
+        If the NewMode field matches the Mode field in an entry of the SupportedModes list but the device is unable to transition as requested, the server SHALL respond with a FAILURE status response.
+        If the NewMode field matches the Mode field in an entry of the SupportedModes list and the device is able to transition as requested the server SHALL set the CurrentMode attribute to the NewMode value and SHALL respond with a SUCCESS status response.
+      </description>
+      <arg name="NewMode" type="INT8U" optional="false"/>
+    </command>
+
+    <command source="client" code="0x01" name="ChangeToModeWithStatus" optional="true">
+      <description>
+        This command is used to change device modes using the same mechanism and semantics as ChangeToMode, and additionally obtaining a response with an ability for clients to determine causes of failures with fine-grained details. For status response depending on NewMode field, see ChangeToMode command.
+        On receipt of this command the device SHALL respond with a ChangeToModeResponse command.
+      </description>
+      <arg name="NewMode" type="INT8U" optional="false"/>
+    </command>
+
+    <command source="server" code="0x02" name="ChangeToModeResponse" optional="true">
+      <description>
+        This command is sent by the device on receipt of the ChangeToModeWithStatus command.
       </description>
-      <arg name="NewMode" type="INT8U" />
+      <arg name="Status" type="ENUM8" optional="false"/>
+      <arg name="StatusText" type="CHAR_STRING" optional="true"/>
     </command>
 
   </cluster>
 
   <bitmap name="ModeSelectFeature" type="BITMAP32">
     <cluster code="0x0050"/>
     <field name="DEPONOFF" mask="0x1"/>
+    <field name="EXT" mask="0x2"/>
```

Comment:
I know the existing thing had DEPONOFF, but feature bitmaps generally use the feature name, not the feature code.  We should align this with that general behavior.